### PR TITLE
201 readme updates

### PR DESCRIPTION
**Pull request by poetw**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: 201/ReadMe.md [hunk 1]
```diff
@@ -1,92 +1,185 @@
...
+    - If you'd like to reuse an existing SP, make sure it has a password secret. You can associate a new secret with your existing SP with these [instructions](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-3-create-a-new-application-secret).
+  
+    Please take note of the Client ID (a.k.a., App ID), tenantID, service principal object ID and Client Secret generated in this step.
 
-* Use the client ID and Client Secret from the first step to create a Kubernetes secret on the connected cluster:
-```
-kubectl create secret generic secrets-store-creds --from-literal clientid="<client-id>" --from-literal clientsecret="<client-secret>"
+2. Use the client ID and client secret as plain text from the first step to create a Kubernetes secret on the connected cluster:
+    ```
+    kubectl create secret generic secrets-store-creds --from-literal clientid="<client-id>" --from-literal clientsecret="<client-secret>"
```
Comment (iwangjintian) [inline 201/ReadMe.md:32]: Looks like we are not feeding the secret name into helm. Please stress that the secret name should not be modified.

File: 201/ReadMe.md [hunk 2]
```diff
@@ -1,92 +1,185 @@
...
+    helm uninstall <char name>
+    ```
+
+    For example:
+    ```bash
+    helm uninstall 201chart-1692064686 
+    ```
+
+## Validation
+A pod with prefix "101chart" should be running in default namespace. Check your pod status with:
```
Comment (RemindD) [inline 201/ReadMe.md:150]: pod name?

**Review by RemindD — APPROVED** [review]

File: 201/ReadMe.md (hunk 1 above)
Comment (poetw) [reply 201/ReadMe.md:32]: Added a comment to step 9

File: 201/ReadMe.md (hunk 2 above)
Comment (poetw) [reply 201/ReadMe.md:150]: Fixed.

**Pull request merged**